draft-lmbdhk-quic-multipath.md

```diff
@@ -689,7 +664,7 @@ under the "QUIC Protocol" heading.
 Value                                              | Frame Name          | Specification
 ---------------------------------------------------|---------------------|-----------------
 TBD-00 - TBD-01 (experiments use 0xbaba00-0xbaba01)| ACK_MP              | {{mp-ack-frame}}
-TBD-02 (experiments use 0xbaba03)                  | PATH_STATUS         | {{path-status-frame}}
+TBD-02 (experiments use 0xbaba03)                  | PATH_ABANDON         | {{path-abandon-frame}}
 {: #frame-types title="Addition to QUIC Frame Types Entries"}
 
```

Comment:
How about not reusing the exact same code as the old PATH_STATUS frame? I would suggest something like "experiments use 0xbaba05".

Reply:
Yes, the PATH_ABANDON type is changed to 0xbaba05 now.